tests/test_contrib.py

```diff
@@ -568,7 +569,7 @@ def test_ivf_train_2level(self):
         # normally 47 / 200 differences
         ndiff = (Iref != Inew).sum()
         self.assertLess(ndiff, 51)
```

Comment:
is there away to relax the threshold such that this test passes in SR mode as well?

Reply:
@mengdilin The test case passes in my machine with a `ndiff` value of `50` (not sure why it fails here). I will do another push that resolves the conflict. I have also updated the name from `avx512-sr` to `avx512_spr` to be consistent with what numpy returns.